contracts/smartwallet/SmartWallet.sol

```diff
@@ -174,10 +160,35 @@ contract SmartWallet is IForwarder {
         //not enough, in that case there might be enough gas to complete the relay and the token payment would be collected
         //For that reason, the next require line must be left uncommented, to avoid malicious relayer attacks to destination contract
         //methods that revert if the gasleft() is not enough to execute whatever logic they have.
-
         require(gasleft() > req.gas, "Not enough gas left");
         (success, ret) = req.to.call{gas: req.gas, value: req.value}(req.data);
 
+        if (req.tokenAmount > 0) {
+            bool successPayment;
+            bytes memory retPayment;
+            if (req.tokenContract == address(0)) {
+                (successPayment, retPayment) = payable(feesReceiver).call{
```

Comment:
Why are you using a call to transfer RBTC instead of a transfer? payable(feesReceiver).transfer(balanceToTransfer); ?

Reply:
According to [solidity by example ](https://solidity-by-example.org/sending-ether/)this is the recommended way.